labgrid/driver/sshdriver.py

```diff
@@ -389,12 +391,28 @@ def get_status(self):
         """The SSHDriver is always connected, return 1"""
         return 1
 
+    def _scp_supports_explicit_sftp_mode(self):
+        version = subprocess.run(["ssh", "-V"], capture_output=True, text=True)
+        version = re.match(r"^OpenSSH_(\d+)\.(\d+)", version.stderr)
+        major, minor = map(int, version.groups())
+
+        # OpenSSH >= 8.6 supports explicitly using the SFTP protocol via -s
+        if major == 8 and minor >= 6:
+            return True
+        # OpenSSH >= 9.0 default to the SFTP protocol
+        if major >=9:
```

Comment:
```suggestion
        if major >= 9:
```
Small nitpick, rest looks good.

Reply:
I'm annoyed I missed that one. It's fixed now.